CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing Guide
+
+## Development
+
+### Python Version
+
+Developing in this repository requires Python 3.10 or higher.
+
+### Set-up
+
+Clone the repo by running:
+
+```bash
+git clone git@github.com:coinbase/cdp-sdk-python.git
+```
+
+To install all dependencies, run:
+
+```bash
+make install-deps
+```
+
+### Linting
+
+To autocorrect all lint errors, run:
+
+```bash
+make lint-fix
+```
+
+To detect all lint errors, run:
+
+```bash
+make lint
+```
+
+### Testing
+
+Install the `pytest` and `pytest-cov` packages to run tests:
```

Comment:
these should be included in the `make install-deps` command

Reply:
ah was it recently added? for some reason i didn't have these so manually installed em. will delete!